configuration/enable-offline-mode.adoc

```diff
@@ -18,8 +18,7 @@ To deploy and operate {product-title} in offline mode:
 
 . Download {product-title-short} images and install them in your clusters. If you are using {ocp}, you can use link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html/operators/understanding-operators#operator-lifecycle-manager-olm[{olm-first}] and OperatorHub to download images to a workstation that is connected to the internet. The workstation then pushes images to a mirror registry that is also connected to your secured cluster. For other platforms, you can use a program such as Skopeo or Docker to pull the images from the remote registry and push them to your own private registry, as described in xref:../configuration/enable-offline-mode.adoc#download-images-for-offline-use[Downloading images for offline use].
 . Enable offline mode during installation.
-. (Optional) Routinely update Scanner’s vulnerability list by uploading a new definitions file.
-. (Optional) When required, add support for runtime collection on more kernel versions by uploading new kernel support packages.
+. Routinely update Scanner’s vulnerability list by uploading a new definitions file.
```

Comment:
Should we provide a recommendation here? If so, let's recommend at least once per day